container-images/jdk17/builder/Dockerfile

```diff
@@ -0,0 +1,349 @@
+
+# This is a Dockerfile for the jboss-eap-8-tech-preview/eap81-openjdk17-builder-openshift-rhel9:1.0.0.Beta image.
+
+
+## START target image jboss-eap-8-tech-preview/eap81-openjdk17-builder-openshift-rhel9:1.0.0.Beta
+## \
+    FROM registry.access.redhat.com/ubi9/ubi-minimal
+
+
+    USER root
+
+###### START module 'jboss.container.user:2.0+jboss1'
+###### \
+        # Copy 'jboss.container.user' module content
+        COPY modules/jboss.container.user /tmp/scripts/jboss.container.user
```

Comment:
Here, what is the build context so that we can `COPY` the `modules/jboss.container.user` directory?